File: java_projects/glue_code/commons-validator/src/test/java/org/apache/commons/validator/NameBean.java
package org.apache.commons.validator;

import org.graalvm.polyglot.Value;

public class NameBean {
  protected String lastName = null;
  protected String middleName = null;
  protected String firstName = null;
  private static Value clz = ContextInitializer.getPythonClass("/NameBean.py", "NameBean");
  private Value obj;

  public NameBean(Value obj) {
    this.obj = obj;
  }

  public Value getPythonObject() {
    return obj;
  }

  public void setLastName(String lastName) {
    //
    // this.lastName = lastName;
    //

    obj.invokeMember("setLastName", lastName);
  }

  public String getLastName() {
    //
    // return lastName;
    //

    // TODO: Check the type mapping below!
    return obj.invokeMember("getLastName").as(String.class);
  }

  public void setMiddleName(String middleName) {
    //
    // this.middleName = middleName;
    //

    obj.invokeMember("setMiddleName", middleName);
  }

  public String getMiddleName() {
    //
    // return middleName;
    //

    // TODO: Check the type mapping below!
    return obj.invokeMember("getMiddleName").as(String.class);
  }

  public void setFirstName(String firstName) {
    //
    // this.firstName = firstName;
    //

    obj.invokeMember("setFirstName", firstName);
  }

  public String getFirstName() {
    //
    // return firstName;
    //

    // TODO: Check the type mapping below!
    return obj.invokeMember("getFirstName").as(String.class);
  }
}

File: data/recomposed_projects/deepseek-coder-33b-instruct/commons-validator/src/test/org/apache/commons/validator/NameBean.py
class NameBean:

    def __init__(self) -> None:

        self._firstName = None

        self._middleName = None

        self._lastName = None

    
    def getFirstName(self) -> str:
        return self._firstName
    

    def setFirstName(self, firstName) -> None:
        self._firstName = firstName

    
    def getMiddleName(self) -> str:
        return self._middleName
    

    def setMiddleName(self, middleName) -> None:
        self._middleName = middleName

    
    def getLastName(self) -> str:
        return self._lastName
    

    def setLastName(self, lastName) -> None:
        self._lastName = lastName
